content/4_Gold/Divis.mdx

```diff
@@ -69,6 +76,7 @@ If you've never encountered any **number theory** before, AoPS is a good place t
 	</Resource>
 	<Resource source="CPH" title="21.1 - Primes & Factors" starred></Resource>
 	<Resource source="PAPS" title="16.1, 16.2 - Number Theory"></Resource>
+	<Resource source="MONT@Modern Olympiad Number Theory" title="1.4, 1.6, 1.7, 1.8, 3.1, 3.2 - Prime Factorization, GCD and LCM, & Divisor Functions" url="https://drive.google.com/file/d/1BcJTLjQaelZ4w_70oHKyImC2I8zLfyrt/view" starred></Resource>
```

Comment:
This is also what I changed

Reply:
Would it be possible to shorten the title? It seems a bit long but if it's difficult to shorten that's fine.